src/components/TodoListEmptyComponent/style.ts

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {StyleSheet, TextStyle, ViewStyle} from 'react-native';
+
+type TodoListEmptyComponentType = {
```

Comment:
Is this type for component or for styles? TodoListEmptyComponentStylesType or TodoListEmptyComponentStylesProps